src/py-opentimelineio/opentimelineio/url_utils.py

```diff
@@ -4,12 +4,14 @@
 """Utilities for conversion between urls and file paths"""
 
 import os
+import re
 
 from urllib import (
     parse as urlparse,
     request
 )
 import pathlib
```

Comment:
Super nitpick request.
Could you please remove this import as well and use "Path" in the `url_from_filepath()` function?

Reply:
Sure, can do!